src/sagemaker/tensorflow/README.rst

```diff
@@ -19,6 +30,27 @@ Suppose that you already have a TensorFlow training script called
 ``tf-train.py``. You can train this script in SageMaker Learner as
 follows:
 
+For Script Mode (only available with TensorFlow version 1.11 or higher)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Script Mode will be used if ``py_version`` is set to ``py3``. To use Python 2 with script mode set ``script_mode`` to
+``py2``. ``checkpoint_path``, ``training_steps``, ``evaluation_steps`` and ``requirements_file`` are deprecated in
+Script Mode. ``model_dir`` is added to specify where the checkpoint and saved model files will be export to during
+training. ``model_dir`` will be passed to your training script as a command line argument. Please make sure your
+training script can handle it.
```

Comment:
Same here, write the documentation focusing in a first time experience and assume that the reader has zero information. The reader should be able to ramp up very quickly reading the docs. He should be able to go back to the docs later for more details.